api/src/error_email/error_email_template.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <title>CV Manager API Error Email</title>
+    <meta name="generator" content="Google Web Designer 15.2.1.0306" />
+    <style id="gwd-text-style">
+      p {
+        margin: 0;
+      }
+      h1 {
+        margin: 0;
+      }
+      h2 {
+        margin: 0;
+      }
+      h3 {
+        margin: 0;
+      }
+    </style>
+    <style>
+      html,
+      body {
+        width: 100%;
+        height: 100%;
+        margin: 0;
+      }
+      body {
+        background-color: transparent;
+        transform: perspective(1400px) matrix3d(1, 0, 0, 0, 0, 1, 0, 0, 0, 0, 1, 0, 0, 0, 0, 1);
+        transform-style: preserve-3d;
+      }
+      body * {
+        transform-style: preserve-3d;
+      }
+    </style>
+  </head>
+  <body>
+    <p>
+      You are receiving this email because your user is marked to receive error emails on the CV Manager Admin page.
+    </p>
+    <br />
+    <p>This error originated in the ##_ENVIRONMENT_## environment CV Manager API</p>
+    <br />
+    <p>Error Message: ##_ERROR_MESSAGE_##</p>
+    <br />
+    <p>Error occurred at: ##_ERROR_TIME_##</p>
+    <br />
+    <p>View this error in Cloud Run logs: <a href="##_CLOUD_RUN_LOGS_LINK_##">rsu-manager-cloud-run-api logs</a></p>
```

Comment:
Lets make this less dependent on Cloud Run. I think any link to any logs would work since it is just a URL.

Reply:
Updated to LOGS_LINK